src/contentscripts/jobs/situational_awareness.js

```diff
@@ -192,6 +201,114 @@ function requestPowerOutagesLayerUpdate() {
 }
 
 /**
+ * Requests an update to the SES teams location layer.
+ */
+function requestSesTeamsLayerUpdate() {
+    console.debug('updating SES teams layer');
+
+    // Get all units
+    let unit = [];
```

Comment:
I think this will be a problem long term.

I suggest we use the contents of filterViewModel.selectedEntities.peek() to pull the filters that are applied. This would need to come from the injected js not the content script.